gradle/compile.gradle

```diff
@@ -36,9 +37,15 @@ tasks.withType(JavaCompile) {
     // while compiling.
     options.compilerArgs = ['-Xlint:-options', '-Xlint:-path']
     options.incremental = true
+    inputs.property('javaVendor') {
+        java7Home ? 'Oracle Corporation' : javaVendor
```

Comment:
For our build I actually know that the target VM's vendor is Oracle. I could use `JavaInstallationProbe` to get that version. Would you prefer to do so or is the current solution fine?